src/main/java/gift/controller/WishController.java

```diff
@@ -0,0 +1,51 @@
+package gift.controller;
+
+import gift.domain.Member;
+import gift.domain.Wish;
+import gift.dto.WishRequest;
+import gift.dto.WishResponse;
+import gift.service.JwtService;
+import gift.service.MemberService;
+import gift.service.WishService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/wishes")
+public class WishController {
+
+    private WishService wishService;
+
+    public WishController(WishService wishService) {
+        this.wishService = wishService;
+    }
+
+
+    @GetMapping
+    public ResponseEntity<List<WishResponse>> getWishes(@LoginMember Member member) {
+        List<WishResponse> wishes = wishService.getWishes(member.getId());
+        return ResponseEntity.ok(wishes);
+    }
+
+    @PostMapping
+    public ResponseEntity<?> addWish(@RequestBody WishRequest wishRequest, @LoginMember Member member) {
```

Comment:
요런 부분도 마찬가지로 구체적으로 타입을 명시해 주는게 좋습니다!

```suggestion
    public ResponseEntity<String> addWish(@RequestBody WishRequest wishRequest, @LoginMember Member member) {
```


예를 들어 ProductController에서도

```
public Object deleteProduct(@PathVariable("id") Long id) {
        // 요청받은 id를 가진 상품을 삭제
        productDAO.delete(id);

        return REDIRECT_URL;
    }
```

```
public String deleteProduct(@PathVariable("id") Long id) {
        // 요청받은 id를 가진 상품을 삭제
        productDAO.delete(id);

        return REDIRECT_URL;
    }
```

1번과 2번 중 당연하게 2번을 사용하는 것 처럼요!